### ISSUE TITLE
Get same prediciton quality as tensorflow torsk

### ISSUE THREAD

**Issue by nmheim** [nmheim/toto#2]
So far the data was not normalized... is that the reason the predictions are not that good?

**Comment by nmheim** [comment]
In addition to normalizing the data, adding a bias vector to the ESNCell improves prediction a lot (commit 9830b5cccf0c19eeed24ef340694b9c8ec917176). Still have to run over the whole dataset to produce some concrete numbers.

**Comment by nmheim** [comment]
Check what the internal TanhRNNCell of pytorch does. There is a possible second bias term for the reservoir layer, which could mean that the input and reservoir layers have two seperate activation functions...

**Comment by nmheim** [comment]
@jamesavery prediction performance should be fixed with 181dc9b0cbcb0ff0b08153716b04baef2590a4a1

![example](https://user-images.githubusercontent.com/29552345/48662370-86ea8f80-ea81-11e8-9bfa-5f940677ae9f.png)
